go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/kong/deck v1.18.1
 	github.com/kong/go-kong v0.38.0
+	github.com/kong/kubernetes-telemetry v0.0.0-20230214134628-61a122724c2b
```

Comment:
Should we make a proper release of `github.com/kong/kubernetes-telemetry` before using it here? I think that would make sense, WDYT?

Reply:
Sure thing. We can tag it an use it here.